### ISSUE TITLE
Studio is missing all images and CSS

### ISSUE THREAD

**Issue by natea** [regisb/openedx-docker#10]
During one of the builds, something happened and now the Studio site is missing all CSS and images. What would I do to restore the site to its full glory? ;)

![welcome___edx_studio](https://user-images.githubusercontent.com/118984/36948506-c818c092-1f90-11e8-8905-94e06c9b79ee.png)

**Comment by regisb** [comment]
Hmmmm, can you try to run `make assets` ?

**Comment by natea** [comment]
Doing that now. Man, does it take forever! Can you tell it just to collectstatic on the CMS/Studio and not the LMS as well?

**Comment by regisb** [comment]
Yes, you can collect assets just for the CMS by running `docker-compose run --rm cms paver update_assets cms --settings=production`

**Comment by natea** [comment]
Thanks for the tip on just collectstatic for the CMS! The `make assets` finished and my Studio is restored to its full glory!
